include/openssl/dh.h

```diff
@@ -94,6 +94,13 @@ DECLARE_ASN1_ITEM(DHparams)
 # define d2i_DHparams_bio(bp,x) ASN1_d2i_bio_of(DH,DH_new,d2i_DHparams,bp,x)
 # define i2d_DHparams_bio(bp,x) ASN1_i2d_bio_of_const(DH,i2d_DHparams,bp,x)
 
+# define d2i_DHxparams_fp(fp,x) (DH *)ASN1_d2i_fp((char *(*)())DH_new, \
+                (char *(*)())d2i_DHxparams,(fp),(unsigned char **)(x))
```

Comment:
maybe break after the define'd name and then the funcall and its parameters can be indented "properly"?
here, and below.